lua/arccw/client/cl_languages/ko.lua

```diff
@@ -0,0 +1,219 @@
+L["arccw.cvar.mult_reloadtime"]           = "재장전 시간"
+L["arccw.cvar.mult_sighttime"]            = "정조준 시간"
+L["arccw.cvar.mult_defaultclip"]          = "기본 탄약"
+L["arccw.cvar.mult_attchance"]            = "랜덤 부착물 확률"
+
+-- ArcCW_Options_Atts
+L["arccw.attdesc1"]                       = "ArcCW는 ACT3와 비슷한 부착물 인벤토리를, CW2와 비슷한 부착물 잠금 시스템을, TFA와 비슷한 제한없이 부착물을 제공합니다."
+L["arccw.attdesc2"]                       = "ACT3 스타일의 부착물 인벤토리 동작에 대한 모든 옵션을 버리십시오."
+L["arccw.cvar.attinv_free"]               = "공짜 부착물"
+L["arccw.cvar.attinv_lockmode"]           = "부착물 잠금"
+L["arccw.cvar.attinv_loseondie.desc"]     = "부착물 소실 모드:\n0 - 비활성화\n1 = 사망 시 제거\n2 = 사망 시 부착물 박스 드랍"
+L["arccw.cvar.attinv_loseondie"]          = "부착물 소실 모드"
+L["arccw.cvar.atts_pickx.desc"]           = "Pick X는 무기별 부착 가능한 부착물에 대한 제한하는 용도입니다.\n0 = 제한 없음."
+L["arccw.cvar.atts_pickx"]                = "Pick X"
+L["arccw.cvar.enable_dropping"]           = "부착물 드랍"
+L["arccw.cvar.atts_spawnrand"]            = "스폰 시 랜덤 부착물"
+L["arccw.cvar.atts_ubglautoload"]         = "하단 부착식 무기 자동 장전"
+L["arccw.blacklist"]                      = "블랙리스트 메뉴"
+
+-- ArcCW_Options_Server
+L["arccw.cvar.enable_penetration"]        = "관통 활성화"
+L["arccw.cvar.enable_customization"]      = "커스터마이징 활성화"
+L["arccw.cvar.truenames"]                 = "트루 네임 (재시작 필요)"
+L["arccw.cvar.equipmentammo.desc"]        = "탄약 종류는 255개로 제한되어 있습니다. 활성화 시 문제가 발생할 수 있습니다. (재시작 필요)"
+L["arccw.cvar.equipmentammo"]             = "무기별 독립 탄약"
+L["arccw.cvar.equipmentsingleton.desc"]   = "싱글톤은 사용 후 인벤토리에서 무기가 제거됩니다. (재시작 필요)"
+L["arccw.cvar.equipmentsingleton"]        = "수류탄 / 장비 싱글톤"
+L["arccw.cvar.equipmenttime"]             = "장비 자폭 시간"
+L["arccw.cvar.throwinertia"]              = "수류탄 초기 가속도"
+L["arccw.cvar.limityear_enable"]          = "연대 제한 활성화"
+L["arccw.cvar.limityear"]                 = "연대 제한"
+L["arccw.cvar.override_crosshair_off"]    = "조준점 강제 비활성화"
+L["arccw.cvar.override_deploychambered"]  = "전개 시 약실 개방"
+L["arccw.cvar.override_barrellength"]     = "총구 벽 가로막힘 활성화"
+L["arccw.cvar.doorbust"]                  = "문 파괴 활성화"
+L["arccw.cvar.desync"]                    = "미동기화식 (Desync) 탄환"
+L["arccw.cvar.desync.desc"]               = "서버와 클라이언트간 탄환의 동기화를 해제합니다. 스프레드 핵을 방지하는데 유용합니다."
+L["arccw.cvar.weakensounds"]              = "낮은 발포음"
+L["arccw.cvar.weakensounds.desc"]         = "모든 무기의 발포음을 60에서 150 데시벨로 감소시킵니다."
+
+-- ArcCW_Options_Dev
+L["arccw.dev_info1"]                      = "부착물과 관련된 부분으로 개발자만이 사용할 수 있습니다."
+L["arccw.dev_info2"]                      = "설정은 저장되지 않습니다."
+L["arccw.cvar.dev_reloadonadmincleanup"]  = "관리자 클린업 시 리로드"
+L["arccw.cvar.dev_reloadonadmincleanup.desc"]    = "기본 값 : 비활성화\n관리자 클린업이 모든 부착물 데이터를 리로드합니다."
+L["arccw.cvar.dev_registerentities"]      = "부착물 엔티티 등록"
+L["arccw.cvar.dev_registerentities.desc"] = "기본 값 : 활성화\nArcCW 부착물의 엔티티를 생성합니다. 부착물 데이터 리로드 시간이 증가합니다."
+L["arccw.cvar.dev_reloadatts"]            = "부착물 데이터 리로드"
+L["arccw.cvar.dev_reloadatts.desc"]       = "부착물 데이터를 리로드합니다. 부착물 수정 후 사용하세요."
+
```

Comment:
Missing `L["arccw.cvar.dev_reloadlangs"]            = "Reload Languages"` and 
`L["arccw.cvar.dev_reloadlangs.desc"]       = "Reloads all language files."`

Reply:
Added new translate texts.